net/tor/src/opnsense/service/scripts/tor/setup.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
double-check permission, can't see from here

Reply:
755 - should be ok